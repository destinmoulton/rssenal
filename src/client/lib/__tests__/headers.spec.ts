import {
    generateJSONHeaders,
    generateJWTHeaders,
    generateJWTJSONHeaders
} from "../headers";

describe("lib/headers.ts", () => {
    it("generateJSONHeaders()", () => {
        const headers = generateJSONHeaders();
        expect(headers).toMatchSnapshot();
    });
});
