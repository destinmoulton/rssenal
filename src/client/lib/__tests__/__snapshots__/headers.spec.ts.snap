// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lib/headers.ts generateJSONHeaders() 1`] = `
Headers {
  Symbol(map): Object {
    "Accept": Array [
      "application/json",
    ],
    "Content-Type": Array [
      "application/json",
    ],
  },
}
`;
